Libraries/Components/View/ReactNativeStyleAttributes.js

```diff
@@ -47,4 +48,16 @@ ReactNativeStyleAttributes.shadowColor = colorAttributes;
 ReactNativeStyleAttributes.textDecorationColor = colorAttributes;
 ReactNativeStyleAttributes.tintColor = colorAttributes;
 
+if (Platform.OS === 'android') {
+  // ReactAndroid doesn't currently have the capability to have variable typing
+  // (eg. string or array) of a native attribute.  To work around that, we can
+  // transform variable types into one standard type, as done here with
+  // textDecorationLine which can be an array of enum values or a space-separated
+  // string containing the same values.
+  var processTextDecorationLine = require('processTextDecorationLine');
```

Comment:
I'm unsure what this does on iOS since the packager is not aware of `Platform.OS` checks and will try to find a module called `processTextDecorationLine` for the iOS JS bundle. 

cc @martinbigio: if a cross-platform file requires an Android-only file (that is, the .ios.js or .js counterparts don't exist), does the packager define what it's behavior is in that scenario? I could see it being reasonable either way to exclude the module from the iOS bundle (so if you tried to require it you'd get a runtime error), or to say the behavior is undefined / always be sure to define a .ios.js stub.